README.md

```diff
@@ -31,6 +31,7 @@ so please check out your distribution's package list to see if KeePassXC is avai
 - Automatic reload when the database changed on disk
 - Browser integration with KeePassXC-Browser using [native messaging](https://developer.chrome.com/extensions/nativeMessaging) for [Mozilla Firefox](https://addons.mozilla.org/en-US/firefox/addon/keepassxc-browser/) and [Google Chrome, Chromium, Vivaldi, or Brave](https://chrome.google.com/webstore/detail/keepassxc-browser/oboonakemofpalcgghocfoadofidjkkk)
 - Synchronize passwords using KeeShare. See [Using Sharing](./docs/QUICKSTART.md#using-sharing) for more details.
+- A [Command Line Interface (keepassxc-cli)](./src/cli/keepassxc-cli.1)
```

Comment:
@SohamG It's actually already mentioned just above (line 25)! Maybe replace that line instead?

Reply:
I didn't spot that, fixed now!